react-native-template-reason/bsconfig.json

```diff
@@ -15,5 +15,11 @@
       "subdirs": true
     }
   ],
-  "bs-dependencies": ["bs-react-native", "reason-react"]
+  "bs-dependencies": [
+    "reason-react",
+    "bs-react-native/reason-react-native"
+  ],
+  "js-post-build": {
+    "cmd": "./node_modules/bs-react-native/git-monorepo-usage-trick"
+  }
```

Comment:
I think we should make some change here so it's work on CI.
And provide tweaks in the README or something.

Reply:
What do you mean by "some changes"? The post build scripts won't work on CI?

Reply:
node_modules/bs-react-native doesn't exist as that's the repo itself. If you are not confortable on handling the situation (which is totally understandable) we can imaging merging your work in another branch than master & continue working from here.

Reply:
The git+https dependency I've added in package.json doesn't work on CI? It works on my Linux machine.

Reply:
Problem is: by doing that, you refer to master branch. If we make a breaking change to something & update the template at the same time, this technique will just not work at all.
And btw, this monorepo trick will never be released so shouldn't be in the template.

Can't you just use `reason-react-native` as a dep & see if that works?

Reply:
Apparently `reason-react-native` is not published on NPM yet. Shouldn't we wait for it to be published before using it?

Reply:
We are using yarn workspace here. So all packages of this repo are simlinked, so those are available as soon as this is run locally. The website package is doing that (master shows "../reason-react-native" but jsx3-migration branch does have `"reason-react-native`") & it's working :)